### ISSUE TITLE
Native implementation check fails

### ISSUE THREAD

**Issue by phated** [medikoo/es6-weak-map#6]
It seems that in latest Chrome, WeakMap isn't implemented to spec, as `Object.prototype.toString.call(WeakMap.prototype) === '[object WeakMap]'` is false and actually results in `'[object Object]'`.  Would you be open to using [`lodash.isnative`](https://www.npmjs.com/package/lodash.isnative) to do this check?

**Comment by medikoo** [comment]
I'm not keen to do this, as this bug breaks some code (at least on my side), also it should be fixed soon in Chrome, Canary get things right already, so it's temporary state.

**Comment by phated** [comment]
@medikoo that's fair but it will continue to be busted in node 0.12 for a looooong time due to its extremely outdated version of v8.  Would you rather that platform gets the shim than use an actual WeakMap?

**Comment by phated** [comment]
cc @jdalton

**Comment by medikoo** [comment]
@phated I can fix `es6-weak-map/is-native-implemented`, so it detects it properly, and then you can do:
```javascript
if (!require('es6-weak-map/is-native-implemented')) {
  var WeakMap = require('es6-weak-map');
}
```

This however will fallback to any native implementation, even those older way more broken (but I'm not sure how big risk is that)

**Comment by phated** [comment]
That would be excellent.

**Comment by medikoo** [comment]
Fixed and published as v1.0.1

**Comment by jdalton** [comment]
:rock:!
